translations/es.po

```diff
@@ -3273,6 +3276,12 @@ msgid ""
 "the event page."
 msgstr ""
 
+#: app/controllers/events/view/videoroom/list.js:44:24
+msgid ""
+"This column shows the original link of the video. We do not recommend to "
+"share this link as users can access it without logging into {{appName}}.com."
+msgstr ""
+
 #: app/controllers/events/view/videoroom/list.js:51:26
 msgid ""
 "You can share the links in this columns with attendees of an event. On event "
```

Comment:
Correct sentence: "You can share the links in this columns with attendees of an event. Registered attendees or ticket holders will be able to access these links."